### PR TITLE
Change Field Requirements on Split Detail Page

### PR DESCRIPTION
**Pull request by jesseproudman**
### Summary
Allows individual fields to be populated on the split details page.

### CODE REVIEW

File: app/views/admin/split_details/edit.html.erb [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+  <%= f.input :description, as: :text, required: @split_detail.description.present?, hint: "Example: Users often get confused during the account creation process when reaching step 2, 'Personal Details'. We are testing two different versions of this page that aim to clear up that confusion." %>
 
-  <%= f.input :hypothesis, as: :text, hint: "Example: By rearranging some of the fields and adding helper text we can improve sign up completion rates and decrease the amount of support tickets we receive." %>
+  <%= f.input :hypothesis, as: :text, required: @split_detail.hypothesis.present?, hint: "Example: By rearranging some of the fields and adding helper text we can improve sign up completion rates and decrease the amount of support tickets we receive." %>
 
-  <%= f.input :assignment_criteria, as: :text, hint: "Example: Assigned when any user enters the first step of the signup flow. Mobile users are not affected." %>
+  <%= f.input :assignment_criteria, as: :text, required: @split_detail.assignment_criteria.present?, hint: "Example: Assigned when any user enters the first step of the signup flow. Mobile users are not affected." %>
```
Comment (smudge) [inline app/views/admin/split_details/edit.html.erb:18]: Won't these all still hit these validations on the SplitDetail model?

https://github.com/Betterment/test_track/blob/master/app/models/split_detail.rb#L10

```
validates :hypothesis, :assignment_criteria, :description, :owner, :location, :platform, presence: true
```

**Review by smudge — CHANGES_REQUESTED** [review]
TAFN

File: app/views/admin/split_details/edit.html.erb [hunk 2]
```diff
@@ -7,15 +7,15 @@
 
 <%= simple_form_for @split_detail, url: admin_split_details_path, method: :put do |f| %>
   <%= f.error :base %>
-  <%= f.input :owner, label: "Owner" %>
-  <%= f.input :platform, collection: Split.platforms.keys %>
-  <%= f.input :location, label: "Location", hint: "Example: Sign Up" %>
+  <%= f.input :owner, label: "Owner", required: @split_detail.owner.present? %>
```
Comment (smudge) [inline app/views/admin/split_details/edit.html.erb:10]: Nice -- now that the model validations are conditional, I'm wondering if these `required:` options ever actually needed to be set, or if the Rails validations were driving the behavior all along.

**Comment by jesseproudman** [comment]
@smudge Specs pass now. :|

**Pull request merged**